/nmea_transform_publisher:
  ros__parameters:
    child_topic: 'laser_frame'
    nmea_ip: 192.168.0.50
    nmea_port: 9095
    parent_topic: 'map'
    qos_overrides:
      parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
      tf:
        publisher:
          depth: 100
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false
